pkgs/development/libraries/webkit2-sharp/default.nix

```diff
@@ -45,7 +45,7 @@ stdenv.mkDerivation rec {
   installFlags = ["GAPIXMLDIR=/tmp/gapixml"];
 
   passthru = {
-    inherit webkitgtk;
+    inherit webkitgtk_4_0;
```

Comment:
This should probably remain so that the user can get the correct ABI version used by this library. Otherwise, they could just use the top-level one.